### PR TITLE
DNN-7169 Created 2 more helpful error messages when trying to reset the password

### PR DESCRIPTION
**Pull request by donker**
More accurate reporting of failures when password reset doesn't work

### CODE REVIEW

File: Website/App_GlobalResources/SharedResources.resx [hunk 1]
```diff
@@ -999,7 +999,13 @@
   <data name="PasswordResetFailed.Text" xml:space="preserve">
     <value>Your new password was not accepted for security reasons. Please make sure to choose a password which does not match a previously used password and which is long and complex enough to meet the sites password complexity requirements.</value>
   </data>
-  <data name="PasswordResetFailed_UserUndefined.Text" xml:space="preserve">
+ <data name="PasswordInadequate.Text" xml:space="preserve">
+    <value>Your new password was not accepted for security reasons. Please make sure to choose a password which is long and complex enough to meet the sites password complexity requirements.</value>
```
Comment (bdukes) [inline Website/App_GlobalResources/SharedResources.resx]: can you please change "sites" to "site's"?

**Comment by bdukes** [comment]
After fixing typo, looks good to me :+1:

**Comment by cathalconnolly** [comment]
I've removed the milestone as assigning PR's to milestones is the PM's decision (i.e. @jbrinkman )

**Comment by donker** [comment]
OK, I thought it was related to risk.

**Comment by cathalconnolly** [comment]
it often is i.e. a large PR that touches on elements that run on each request (such as CDF) will be assigned to a "large" release such as 8.0.0 to ensure that we have a longer test cycle, or a change in a popular API (such as tabcontroller) may not be taken too close to codefreeze, but we also use it to organise what goes in each release i.e. Joe assigns items to a release, and Charles and I process them. In this case 7.4.2 is effectively codefrozen so when Joe reviews this I would expect it to be assigned to 8.0.0, unless he deems it low-risk (and important enough to come after the codefreeze) - we expect we may take an occasional additional PR but that's up to PM to decide

**Pull request merged**